lte/gateway/c/session_manager/SessionState.h

```diff
@@ -84,7 +84,35 @@ class SessionState {
     std::vector<PolicyRule> dynamic_rules;
     std::vector<PolicyRule> gy_dynamic_rules;
     optional<AggregatedMaximumBitrate> ambr;
+    // 5G specific extensions
+    std::vector<SetGroupPDR> Pdr_rules_;
+    std::vector<SetGroupFAR> Far_rules_;
+    magma::lte::Fsm_state_FsmState   state;
+    std::string sess_id;
+    uint32_t  ver_no;
+    NodeId   nodeId;
+    FSid     Seid;
+    // 5G specific extension routines
+    magma::lte::Fsm_state_FsmState get_smf_state(struct SessionInfo);
+    // To get sess id
+    std::string get_sess_id (struct SessionInfo);
+    // To get sess version no
+    uint32_t  get_sess_ver_no (struct SessionInfo);
+    // To get the Nodeid
+    NodeId get_node_id (struct SessionInfo);
+        // To get the Fseid
+    FSid  getFSeid (struct SessionInfo);
+     // To get the PDR list
+    std::vector<SetGroupFAR> get_far_rule_list(struct SessionInfo);
```

Comment:
This is more of a preference but `get_far_rules` should suffice

Reply:
Changed